app/views/parks/show.html.erb

```diff
@@ -33,7 +33,9 @@
 
 <div>
   <%= link_to 'Update', edit_park_path %>
+  <% if !current_user.nil? && current_user.admin? %>
```

Comment:
Try to avoid mixing negative (!) and positive checks.. Can change to:
`if current_user.present? && current_user.admin?`